.gitlab-ci.yml

```diff
@@ -104,6 +104,7 @@ unit-bs:
       - master
       - tags
   stage: browserstack
+  resource_group: browserstack
```

Comment:
Could we remove [the `waitForAvailability` feature](https://github.com/DataDog/browser-sdk/blob/master/scripts/bs-wrapper.js#L14) then?

Reply:
I would keep it... it doesn't cost much, and it could avoid issues when tests are manually run.